.travis.yml

```diff
@@ -47,7 +47,7 @@ before_install:
 - export PATH="$HOME/miniconda/bin:$HOME/miniconda/lib:$PATH";
 - hash -r;
 - conda config --set always_yes yes --set changeps1 no;
-- conda create -y -q -n test_cq -c pythonocc -c oce -c conda-forge -c dlr-sc pythonocc-core=0.18
+- conda create -y -q -n test_cq -c pythonocc -c oce -c conda-forge -c dlr-sc pythonocc-core=0.18.1
```

Comment:
The .travis.yml file on the adam-urbanczyk-OCC-version-update branch pulls pythonocc-core 0.18.2. Should we standardize on a version, or does it matter?

Reply:
Yes that is on purpose. I wan't to keep 0.18.1 explicitly for now (that is what master used to be tested with). Once the issues with 0.18.2 (Mac) are solved we can switch to that version.